sceptre/cli/helpers.py

```diff
@@ -36,8 +39,9 @@ def decorated(*args, **kwargs):
         try:
             return func(*args, **kwargs)
         except (SceptreException, BotoCoreError, ClientError, Boto3Error,
-                TemplateError) as error:
-            write(error)
+                TemplateError):
+            if "DEBUG_ERRORS" in os.environ:
+                traceback.print_exc()
```

Comment:
Why wouldn't we just always want to do this? I don't know why we'd want to buy the stack track ever? Whenever there IS an error... it'd always be useful to know where the error is coming from. That's just my own two cents.

Reply:
> Why wouldn't we just always want to do this?

@jfalkenstein The pattern has generally been to capture specific exceptions and print relative message in a normal run.  The original plan for this PR was to print the exception when `--debug` was present, and possibly also add logic to capture the original exception that was causing Alex pain in the source issue, like has been done for other exceptions.

@alexharv074 this PR was close with printing the exception when `--debug` was present, but it has drifted into printing the exception when a specific env var is set.  I would prefer to see the exception with printed with `--debug` rather than an env var, as IMO setting/unsetting the env var is more clunky.

Reply:
@craighurley How would you make the logger available inside this decorator function?